### ISSUE TITLE
Cannot run Placenote SDK for iOS 12!

### ISSUE THREAD

**Issue by ghost** [Placenote/PlacenoteSDK-Unity#47]
<img width="866" alt="screenshot 2018-11-01 at 2 24 28 pm" src="https://user-images.githubusercontent.com/30992770/47836136-ff92f000-dde1-11e8-963a-a6570405e637.png">
Hi,
I am trying to build the example scenes from the SDK  and i cannot go through building process in XCode, it gives me this error.
error: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS12.1.sdk/Plugins/iOS/Placenote.framework: No such file or directory

I am using Unity 2018.3 and Xcode 10.1 and iOS 12.1

**Comment by y22ma** [comment]
Just wondering if you can find the Placenote.framework file under Assets/Placenote/Plugins/iOS/

**Comment by ghost** [comment]
@y22ma Yes!

**Comment by pmukherj** [comment]
Hey @syedzainn. Come find us on Slack (https://placenote.com/slack) for faster answers :). 
To clarify, what is the size of that file? It should be upwards of 7MB. Sometimes because of git-lfs, it doesn't download the file when you clone normally.

**Comment by ghost** [comment]
Hi @pmukherj , the file size of placenoteSDK.unitypackage is 12.4 MB

**Comment by pmukherj** [comment]
Hey @syedzainn, I meant the file under Assets/Placenote/Plugins/iOS/ as y22ma pointed out. Just checking what is the size of that file?

P.S: Again, come find us on https://placenote.com/slack for a quicker conversation. Placenote SDK works fine on iOS 12 :). Just have to work out how to get it downloaded properly on your computer!

**Comment by ray8564** [comment]
I have solved my problem just like you @syedzainn.  I misunderstood the error description before. 
 And I discovered that there isn't any Plugins folder in my Xcode.app , so **the solution of this error is that copy the Plugins folder from /Assets/Placenote of your Unity project folder, and paste it to /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS12.1.sdk/.  After that, open the Unity-iPhone.xcodeproj again, and the build will succeed.**  Hope this solution can help someone who meets the same error.

**Comment by uzzi1234** [comment]
Hello i had the same problem with Xcode just delete the placenote framework link it has. then search for it in the list and re addd it and it works.

**Comment by CHaP235** [comment]
I have the same linkage issue as described by @syedzainn. I also did an Unity 2018.3.x update as well to the latest Placenote SDK (Maybe something went wrong? I removed the old files before handed).

My solution is to remove the 'Embedded Binaries' Placenote entry and re-add the Placenote framework inside the builded library folder via reference.

Any solutions or hints to solve this issue would be great. We are using Cloud Services to build the project and at this point these services are failing.
